pkg/u8g2/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=u8g2
 PKG_URL=https://github.com/olikraus/u8g2
-PKG_VERSION=7346987f4e2eadc2e0791b77a1c1c8a27bd72a93
+PKG_VERSION=e8764381a751c1743c01e9f9ea24fb812ab754d5
```

Comment:
also works for newer version, please pump to `9833db01f3e8b865af94e40237fa8c937b732159`